app/controllers/test/components/mentoring/inbox_controller.rb

```diff
@@ -6,4 +6,109 @@ def show
       { value: 'student', label: 'Sort by Student' }
     ]
   end
+
+  def tracks
```

Comment:
I'd argue this is less readable as the file is too long. Unless we move the results to a YAML file or something like that?

Reply:
I think it would make most sense for this controller to use the actual serializer. Having all this mocked data here means that we could change the serializer and this test would still pass but the actual component would then break. Using the real serializer here (and setting the data up via a factory) gives us much safer test coverage.

I think when we do this, this controller will then become much much smaller.

Reply:
I like the above suggestion!

Reply:
Yes. That'll be the best :)

Reply:
So we'll add the serializer support later once Jeremy's PR has been merged, right?

Reply:
Yes, I'll do that later today :)